src/components/atoms/Trustpilot/Stars/Stars.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { axe } from 'jest-axe';
+import React from 'react';
+import { render } from '@testing-library/react';
+import Stars from './Stars';
+import 'jest-styled-components';
+
+describe('<TrustpilotStars />', () => {
+  it('renders default 5 stars rating without a11y violations', async () => {
+    const { container } = render(<Stars />);
+    const results = await axe(container.innerHTML);
+    expect(results).toHaveNoViolations();
+    expect(container).toMatchSnapshot();
+  });
+
+  it.each`
+    rating
+    ${0}
+    ${1}
+    ${1.5}
+    ${2}
+    ${2.5}
+    ${3}
+    ${3.5}
+    ${4}
+    ${4.5}
+    ${5}
+  `('renders without a11y violations the given rating: $rating', async ({ rating }) => {
+    const { container } = render(<Stars rating={rating} />);
+    const results = await axe(container.innerHTML);
+    expect(results).toHaveNoViolations();
+    expect(container).toMatchSnapshot();
```

Comment:
This is generating a super long snapshot. And I don't think `expect(results).toHaveNoViolations();` is necessary for every loop.

I think that's a clear use case for [rerender](https://testing-library.com/docs/react-testing-library/api/#rerender) and maybe we can test if there's a new shape instead of snapshots. WDYT?